src/OpenTelemetry.Exporter.Console/ConsoleExporterMetricsExtensions.cs

```diff
@@ -97,14 +97,11 @@ public static MeterProviderBuilder AddConsoleExporter(this MeterProviderBuilder
 
             var metricExporter = new ConsoleMetricExporter(exporterOptions);
 
-            if (metricReaderOptions.MetricReaderType == (MetricReaderType)(-1))
-            {
-                metricReaderOptions.MetricReaderType = MetricReaderType.Manual;
-            }
-
             var metricReader = metricReaderOptions.MetricReaderType == MetricReaderType.Manual
```

Comment:
I'm considering whether we should just get rid of `MetricReaderType.Manual`? My vote is yes and we use the periodic reader with an infinite interval where appropriate. This idea originated from this suggestion https://github.com/open-telemetry/opentelemetry-specification/pull/2378#discussion_r815327718

Reply:
I would vote for getting rid of "Manual" at least for now, because:
1. The word "Manual" might be misleading. E.g. one could build a reader/exporter that exports metrics based on network connectivity (e.g. only exports data when running on Wifi or LAN), battery status (e.g. only exports data when batter is >50% or the device is charging).
2. If later we want "Manual", it can be an additive change.

Reply:
This PR fixes the immediate bug. I'll remove `MetricReaderType` in a follow up PR.